lib/publiccloud/instance.pm

```diff
@@ -619,23 +621,25 @@ sub measure_boottime() {
 
 Save data collected with measure_boottime in a DB;
 Mainly stored on a remote InfluxDB on a Grafana server.
-
+To activate boottime push, shall be available results and
+  PUBLIC_CLOUD_PERF_PUSH_DATA true/not 0 and
+  _SECRET_PUBLIC_CLOUD_PERF_DB_TOKEN and PUBLIC_CLOUD_PERF_DB_URI defined
 =cut
 
 sub store_boottime_db() {
     my ($self, $results) = @_;
-    return unless (get_var('PUBLIC_CLOUD_PERF_PUSH_DATA') && $results);
-
-    my $url = get_var('PUBLIC_CLOUD_PERF_DB_URI');
+    my $data_push = get_var('PUBLIC_CLOUD_PERF_PUSH_DATA', 1);
+    my $url = get_var('PUBLIC_CLOUD_PERF_DB_URI', 'http://publiccloud-ng.qa.suse.de:8086');
+    my $org = get_var('PUBLIC_CLOUD_PERF_DB_ORG', 'qec');
+    my $db = get_var('PUBLIC_CLOUD_PERF_DB', 'perf_2');
     my $token = get_var('_SECRET_PUBLIC_CLOUD_PERF_DB_TOKEN');
+
+    return unless ($results && $data_push);
     unless ($url && $token) {
```

Comment:
this unless is not relevant anymore

Reply:
Sorry, what of the 2?
the first one still is needed, when results not produced for any error or when we wanted to disable measures

Reply:
ok , so I was wrong we can not and should not fully remove this `unless` .

`$token` is still may be absent . And actually this is how we are handling clone of job into none-production openQA instance . So let's keep unless

BUT we should drop `$url` from it ( from now on it will be ALWAYS defined )

Reply:
Yes, me too, I was thinking that url check no more needed.